deployment/Dockerfile.debian

```diff
@@ -1,6 +1,4 @@
-ARG IMG=debian
-ARG TAG=buster
-FROM $IMG:$TAG
+FROM debian:bookworm-slim
```

Comment:
This breaks CI builds for different Debian versions.

Reply:
Should i revert this and only changed the tag to bookworm-slim ?

Reply:
You would want to adjust the CI pipeline
you can also adjust the default options but the dockerfile must still accept the image and tag as parameters

Reply:
Changing this ?
https://github.com/jellyfin/jellyfin-media-player/blob/ee0717ea933f9bef25cdac6e12d873281d14cc00/.github/workflows/main.yml#L153